api/src/main/java/org/pmiops/workbench/cohortbuilder/SearchGroupItemQueryBuilder.java

```diff
@@ -76,7 +76,7 @@
       "is_standard = %s and concept_id in (select distinct c.concept_id\n"
           + "from `${projectId}.${dataSetId}.cb_criteria` c\n"
           + "join (${childLookup}) a\n"
-          + "on (c.path like concat('%%.', a.id, '.%%') or c.path like concat('%%.', a.id) or c.path like concat(a.id, '.%%') or c.path = a.id)\n"
+          + "on (c.path like concat('%%.', a.id, '.%%') or c.path like concat('%%.', a.id))\n"
```

Comment:
All of your SQL ends with \n. I would probably replace that with a "space"

Reply:
space won't do a carriage return.